markdown.dtx

```diff
@@ -26378,9 +26384,10 @@ local function check_continuation_indentation(s, i, indent_table, is_optional, i
     return true
   end
 
-  local passes, new_index, current_trail = traverse_indent(s, i, indent_table, is_optional, is_blank)
+  local passes, new_index, current_trail, is_cont_line = traverse_indent(s, i, indent_table, is_optional, is_blank)
 
   if passes then
+    indent_table.continues = is_cont_line
```

Comment:
`is_cont_line` and `continues` are confusing names for what is a table of current indents.

```suggestion
  local passes, new_index, current_trail, curr_indent = traverse_indent(s, i, indent_table, is_optional, is_blank)

  if passes then
    indent_table.curr_indent = curr_indent
```